reconcile/utils/saasherder.py

```diff
@@ -1228,7 +1327,9 @@ def publish_promotions(self, success, saas_files, mr_cli):
                     # publish to state to pass promotion gate
                     state_key = f"promotions/{channel}/{commit_sha}"
                     value = {
-                        'success': success
+                        'success': success,
+                        'saas_file': item["saas_file"],
+                        TARGET_CONFIG_HASH: item.get(TARGET_CONFIG_HASH)
```

Comment:
nit: this could be outside of the loop since `value` does not depend on the channel

Reply:
you are right, fixed.